lib/child_process.js

```diff
@@ -276,33 +275,25 @@ ObjectDefineProperty(exec, promisify.custom, {
  * @returns {ChildProcess}
  */
 function execFile(file, args = [], options, callback) {
```

Comment:
I can't decide. My hesitance is because it will be different from `fork()` if we do that and it seems like they should all have the same default value.

Reply:
We could also make it default to `null` on `fork()` if that helps 😅 no strong feelings either way

Reply:
In my opinion, if it is meant to be an array then the default should be an empty array.

Reply:
I'm going to leave it as-is for now. We can always revisit later.